fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -27,6 +27,10 @@ int32_t ineg(int32_t l) {
     return -l;
 }
 
+int64_t lneg(int64_t l) {
+    return -l;
```

Comment:
One problem here is that evaluating `- INT64_MIN` is undefined behaviour in C++. An alternative that works is:

```suggestion
    return l == INT64_MIN ? INT64_MIN : -l;
```